src/materials/DTO/get-material.dto.ts

```diff
@@ -0,0 +1,8 @@
+import { material } from '@prisma/client';
+import { BaseRespondDto } from '../../common/DTO/base-respond.dto';
+
+export class GetMaterialRespondDto extends BaseRespondDto {
+  data: {
+    material: material;
```

Comment:
Do not returns a entity in database to the client. Use the entity to construct a DTO, with only useful fields according to API.yml.